src/containers/shared/components/Transaction/EscrowCreate/parser.ts

```diff
@@ -1,6 +1,22 @@
-import formatAmount from '../../../../../rippled/lib/txSummary/formatAmount'
-import { convertRippleDate } from '../../../../../rippled/lib/utils'
+import { unix } from 'moment'
 
+const EPOCH_OFFSET = 946684800
+const convertRippleDate = (date: any) =>
+  unix(date + EPOCH_OFFSET)
+    .utc()
+    .format()
+
+const formatAmount = (d: any) =>
```

Comment:
There is a formatAmount function already

Reply:
running into dependency cycle issue if I import that function from rippled

Reply:
I think the issue is with convertRippledDate.